apps/mobile/src/libs/components/screen-wrapper/screen-wrapper.tsx

```diff
@@ -25,7 +28,9 @@ const ScreenWrapper: React.FC<Properties> = ({
 		paddingTop: edges.includes("top") ? insets.top : ZERO_PADDING,
 	};
 
-	return <View style={[globalStyles.flex1, paddingStyle]}>{children}</View>;
+	return (
+		<View style={[globalStyles.flex1, paddingStyle, style]}>{children}</View>
+	);
 };
 
```

Comment:
Its a good idea to add style to the wrapper!
And what about  adding a Direction enum for directions mentioned in this component?
```ts
const Direction = {
	BOTTOM: "bottom",
	LEFT: "left",
	RIGHT: "right",
	TOP: "top",
} as const;
```